gradle.properties

```diff
@@ -1,14 +1,12 @@
 groupId=org.eclipse.dataspaceconnector
 defaultVersion=0.0.1-SNAPSHOT
 javaVersion=11
-
```

Comment:
These empty lines in my opinion were really helpful to improve readability, I would reintroduce them

Reply:
they must have gotten removed by IntelliJ